tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -0,0 +1,128 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { assert } from 'chai';
+import { test } from 'mocha';
+import { e2eContainer } from '../../inversify.config';
+import { CLASSES, TYPES } from '../../inversify.types';
+import { Editor } from '../../pageobjects/ide/Editor';
+import { GitPlugin } from '../../pageobjects/ide/GitPlugin';
+import { Ide } from '../../pageobjects/ide/Ide';
+import { ProjectTree } from '../../pageobjects/ide/ProjectTree';
+import { QuickOpenContainer } from '../../pageobjects/ide/QuickOpenContainer';
+import { ICheLoginPage } from '../../pageobjects/login/ICheLoginPage';
+import { TestConstants } from '../../TestConstants';
+import { DriverHelper } from '../../utils/DriverHelper';
+import { NameGenerator } from '../../utils/NameGenerator';
+import { CheGitApi } from '../../utils/VCS/CheGitApi';
+import { GitHubUtil } from '../../utils/VCS/github/GitHubUtil';
+import { TestWorkspaceUtil } from '../../utils/workspace/TestWorkspaceUtil';
+import { TopMenu } from '../../pageobjects/ide/TopMenu';
+
+
+
+const driverHelper: DriverHelper = e2eContainer.get(CLASSES.DriverHelper);
+const ide: Ide = e2eContainer.get(CLASSES.Ide);
+const quickOpenContainer: QuickOpenContainer = e2eContainer.get(CLASSES.QuickOpenContainer);
+const editor: Editor = e2eContainer.get(CLASSES.Editor);
+const namespace: string = TestConstants.TS_SELENIUM_USERNAME;
+const workspaceName: string = TestConstants.TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME;
+const topMenu: TopMenu = e2eContainer.get(CLASSES.TopMenu);
+const loginPage: ICheLoginPage = e2eContainer.get<ICheLoginPage>(TYPES.CheLogin);
+const gitHubUtils: GitHubUtil = e2eContainer.get<GitHubUtil>(CLASSES.GitHubUtil);
+const cheGitAPI: CheGitApi = e2eContainer.get(CLASSES.CheGitApi);
+const projectTree: ProjectTree = e2eContainer.get(CLASSES.ProjectTree);
+const gitPlugin: GitPlugin = e2eContainer.get(CLASSES.GitPlugin);
+const testWorkspaceUtils: TestWorkspaceUtil = e2eContainer.get<TestWorkspaceUtil>(TYPES.WorkspaceUtil);
+
+
+suite('Git with ssh workflow', async () => {
+    const workspacePrefixUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/dashboard/#/ide/TestConstants.TS_SELENIUM_USERNAME/`;
+    const wsNameCheckGeneratingKeys = 'checkGeneraringSsh';
+    const wsNameCheckPropagatingKeys = 'checkPropagatingSsh';
+    const committedFile = 'README.md';
+
+    suiteSetup(async function () {
+        let wsConfig = await testWorkspaceUtils.getBaseDevfile();
+        wsConfig.metadata!.name = wsNameCheckGeneratingKeys;
+        await testWorkspaceUtils.createWsFromDevFile(wsConfig);
+    });
+
+    test('Login into workspace and open tree container', async () => {
+        await driverHelper.navigateToUrl(workspacePrefixUrl + wsNameCheckGeneratingKeys);
+        await loginPage.login();
+        await driverHelper.navigateToUrl(workspacePrefixUrl + wsNameCheckGeneratingKeys);
```

Comment:
This line seems redundant to me. If login is successful, user should be redirected to the URL that was pasted before a login. Am I wrong?

Reply:
You are right - has been removed - thanks